README.md

```diff
@@ -9,4 +9,11 @@ This documentation is made up of various [arc42 structured](https://docs.arc42.o
 - [Overview](./systems/overview/)
 - [WikibaseRepo](./systems/WikibaseRepo/)
 - [WikibaseClient](./systems/WikibaseClient/)
-- TBA...
+- WikibaseLexeme
+- EntitySchema
+- QualityConstraints
+- Article Placeholder
+- Property Suggester
+- Termbox SSR
+- Query UI
+- Query Builder
```

Comment:
Hm. We've had this discussion a few times now, and I still don't see these on the same level. Except maybe for the query service, none of the new additions even fit the *system* label IMO.

Reply:
Yeah, did we write those notes down anywhere? I don't see them in the Google doc...

Reply:
No, I don't think we did. Ughhhh.

Reply:
Yes, from what I remember, we agreed that the top components are repo, client and query service.
Everything else is at least one level down from there.